server/src/actors/project.rs

```diff
@@ -222,17 +215,43 @@ impl Default for ProjectConfig {
             allowed_domains: vec!["*".to_string()],
             trusted_relays: vec![],
             pii_config: None,
-            exclude_fields: vec![],
             grouping_config: None,
-            scrub_ip_addresses: false,
-            sensitive_fields: vec![],
-            scrub_defaults: false,
-            scrub_data: false,
             filter_settings: FiltersConfig::default(),
+            datascrubbing_settings: DataScrubbingConfig::default(),
+            datascrubbing_pii_config: AtomicLazyCell::new(),
         }
     }
 }
 
+impl ProjectConfig {
+    /// Get the PII config derived from datascrubbing settings.
+    fn get_datascrubbing_pii_config(&self) -> Option<&PiiConfig> {
```

Comment:
As per [naming convention](https://rust-lang-nursery.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter), this shouldn't have a `get_` prefix. 

Following the above comment, I would suggest to make this a member of `DataScrubbingConfig`, however, and rename to `to_pii_config`.